### DISCUSSION TITLE
ONNX Runtime on GPU of an Android System

### DISCUSSION THREAD

**Discussion by besartgrabanica** [microsoft/onnxruntime#6693] [Other Q&A]
Hello,

Is it possible to do the inference of a model on the GPU of an Android run system? The model has been designed using PyTorch.

Thanks in advance!

**Comment by lenisha** [comment]
@manashgoswami  - could you pls take a look?

**Reply by guoyu-wang** [reply]
https://github.com/microsoft/onnxruntime/issues/6694 is tracking this, 

> We have the NNAPI execution provider which uses CPU/GPU/NPU for model execution on Android.
> Please see [NNAPI Execution Provider](https://www.onnxruntime.ai/docs/reference/execution-providers/NNAPI-ExecutionProvider.html)

**Comment by faxu** [comment]
Cross-posted here: https://github.com/microsoft/onnxruntime/issues/6694
